doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -322,6 +322,7 @@ pimcore:
 - Removed deprecated property `Pimcore\Model\Asset::$types`, use `getTypes()` instead
 - Removed AdminSessionHandler and AdminSessionListener. The session is now handled by Symfony.
 - Methods `Pimcore\Navigation::setDefaultPageType`, `Pimcore\Navigation::getDefaultPageType`, `Pimcore\Navigation\Container::_sort() and `Pimcore\Navigation\Page::_normalizePropertyName()` have been marked as internal.
+- Added a migration to regenerate `classes`, `fieldCollections`, `objectBricks` and `custom layouts`. Make sure to run `bin/console doctrine:migration:exec 'Pimcore\Bundle\CoreBundle\Migrations\Version20230412105530'` when upgrading to Pimcore 11.
```

Comment:
The migration looks and runs fine, just please move this line to the [preparation guide](https://github.com/pimcore/pimcore/blob/11.x/doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md).

Reply:
added separate notes to the upgrade guide see https://github.com/pimcore/pimcore/pull/14903/commits/ece4dc66b89a7a33d536ceeecf8655fd19d99d74

Reply:
thank you, but imho we should remove this line from the upgrade notes as well
```suggestion
```